demo/src/app/components/popover/demos/autoclose/popover-autoclose.html

```diff
@@ -0,0 +1,85 @@
+<p>
```

Comment:
I would change the layout for this:
![screen shot 2018-07-31 at 10 25 24](https://user-images.githubusercontent.com/8074436/43447773-82015d4e-94ac-11e8-844d-1342404a36b7.png)

- buttons stick to text below them
- 2nd example closes on inside click → it triggers 1st example to open right away → confusing
- last example: two buttons are stuck to each other
- code example is super-long (not sure what to do with it... reformat? simplify?)

Reply:
I agree it's long, especially the code sample. Let me rethink it to simplify, and I'll go through your points then.